packages/manager-components/src/components/clipboard/clipboard.component.tsx

```diff
@@ -0,0 +1,40 @@
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { OdsClipboardAttribute } from '@ovhcloud/ods-components';
+import { OsdsClipboard, OsdsText } from '@ovhcloud/ods-components/react';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import './translations';
+
+export const Clipboard: React.FC<OdsClipboardAttribute> = ({
+  disabled,
+  inline,
+  value,
+}) => {
+  const { t } = useTranslation('clipboard');
+
+  return (
+    <OsdsClipboard
+      value={value}
+      inline={inline}
+      disabled={disabled}
```

Comment:
```suggestion
export const Clipboard: React.FC<OdsClipboardAttribute> = (props) => {
  const { t } = useTranslation('clipboard');

  return (
    <OsdsClipboard
      {...props}
```

If they add or remove props, you won't have to change the code